src/server/master_server/BUILD.bazel

```diff
@@ -0,0 +1,10 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
+
+cc_library(
+    name = "chunk_server_manager",
+    srcs = ["chunk_server_manager.cc"],
+    hdrs = ["chunk_server_manager.h"],
+    visibility = ["//visibility:public"],
+    deps = ["@com_google_absl//absl/container:flat_hash_map",
```

Comment:
nit. can we format it a bit to be
```
cc_library(
     name = "chunk_server_manager",
     srcs = ["chunk_server_manager.cc"],
     hdrs = ["chunk_server_manager.h"],
     visibility = ["//visibility:public"],
     deps = [
         "//src/protos:cc_chunk_server_proto",
         "@com_google_absl//absl/container:flat_hash_map",
    ],
 )
```

Reply:
I don't see it being resolved? Is this some IDE auto formatting that folds it back again?

Reply:
just pushed a change for this. Check my latest commit

Reply:
You meant this?
https://github.com/Michael-Tu/cppGFS2.0/commit/fc1a68eb7c0e72af24d7ee2e08914042b20a94d6

The before/after looks the same

Reply:
Okay?